src/lib/components/tablev2/SingleSelectableContent.js

```diff
@@ -134,11 +150,24 @@ export default function SingleSelectableContent({
     ],
   );
 
+  React.useEffect(() => {
+    if (listRef.current) {
+      const outerRef = listRef.current._outerRef;
```

Comment:
I think the approach is good I just wonder why they prefixed outerRef with an underscore. I makes me feel that the lib want to keep this field private to be able to evolve it (to be checked). I would better not use it if they really want to keep it private. To workaround this we can put a ref on the parent and another one on one of the child

Reply:
well it's definitly a 'private' props, as it is not in the doc. But we need to access to this particular dom element since it's where is the scrollBar.
We could get a ref from the parent element and get the element with `.firstchild` but the same issue arise, what if the library change, and add more elements between the scroll container and the parent ?